public/api-spec.json

```diff
@@ -506,14 +506,6 @@
         "summary": "Add a user role assignment.",
         "operationId": "addUserRole",
         "parameters": [
-          {
```

Comment:
There seem like a lot of changes in this file that don't change anything about the structure of the API. Is this generated by a tool? Do we know why the order of fields changed?

Reply:
Oh man, that's really too bad. Thanks for the context.